website/docs/advanced/feature-toggle-types.md

```diff
@@ -33,7 +33,7 @@ Here's the list of the feature toggle types that Unleash supports together with
 
 You can mark feature toggles as `stale`. This is a way to deprecate a feature toggle without removing the active configuration for connected applications. Use this to signal that you should stop using the feature in your applications. Stale toggles will show as stale in the ["technical debt dashboard"](../user_guide/technical_debt).
 
-When you mark a toggle as stale, Unleash will emit an event. You can use [an addon](https://docs.getunleash.io/addons/index) to integrate this with your systems, for instance to post a message in a Slack channel.
+When you mark a toggle as stale, Unleash will emit an event. You can use [an addon](/addons) to integrate this with your systems, for instance to post a message in a Slack channel.
```

Comment:
Nice catch! Changing it to the internal link is definitely better. However, we should probably link to the markdown file instead of to the final link:

```suggestion
When you mark a toggle as stale, Unleash will emit an event. You can use [an addon](../addons/addons.md) to integrate this with your systems, for instance to post a message in a Slack channel.
```

Reply:
Thanks, TIL! :smile: